builtins/web/structured-clone.cpp

```diff
@@ -45,6 +45,8 @@ JSObject *ReadStructuredClone(JSContext *cx, JSStructuredCloneReader *r,
   return params_obj;
 }
 
+DEF_ERR(CloningFailed, JSEXN_TYPEERR, "The object could not be cloned", 0)
```

Comment:
I believe this is supposed to be a `DataCloneError` DOM exception per https://html.spec.whatwg.org/multipage/structured-data.html#structured-cloning.